src/pages/register.js

```diff
@@ -0,0 +1,71 @@
+import { useEffect, useState } from "react";
+import './register.css';
+
+const Register = (props) => {
+
+  //State variables
+  const [walletAddress, setWallet] = useState("");
+  const [status, setStatus] = useState("");
+  const [description, setDescription] = useState("");
+  const [name, setName] = useState("");
+  const [url, setURL] = useState("");
+
+  const connectWalletPressed = async () => { //TODO: connect to wallet
+   
+  };
+
+  const onMintPressed = async () => { //TODO: mint profile token
+    
+  };
+
+  return (
+    <div className="Minter">
+      <button id="walletButton" onClick={connectWalletPressed}>
+        {walletAddress.length > 0 ? (
+          "Connected: " +
+          String(walletAddress).substring(0, 6) +
```

Comment:
Is the casting to String really needed? Based on the previous use of `walletAddress`, it seems to be a string already.

Reply:
Not sure about it. I think it'll depend on what data we extract from web3.js/ether.js object. Removing the casting for now.